--- a/init_scripts/perfsonar-configure_nic_parameters
+++ b/init_scripts/perfsonar-configure_nic_parameters
@@ -20,10 +20,10 @@
 BANDWIDTH_ENABLED=
 NDTNPAD_ENABLED=
 
-NDT=`/sbin/chkconfig --list ndt | grep on`
-NPAD=`/sbin/chkconfig --list npad | grep -w on`
-OWAMP=`/sbin/chkconfig --list owamp-server | grep -w on`
-BWCTL=`/sbin/chkconfig --list bwctl-server | grep -w on`
+NDT=`/sbin/insserv -s | grep '^S:.*:ndt$'`
+NPAD=`/sbin/insserv -s | grep '^S:.*:npad$'`
+OWAMP=`/sbin/insserv -s | grep '^S:.*:owamp-server$'`
+BWCTL=`/sbin/insserv -s | grep '^S:.*:bwctl-server$'`
 
 if [ "x$NDT" != "x" -o "x$NPAD" != "x" ]; then
         NDTNPAD_ENABLED=1
--- a/scripts/system_environment/disable_unwanted_services
+++ b/scripts/system_environment/disable_unwanted_services
@@ -8,7 +8,7 @@
     if type systemctl &>/dev/null; then
         systemctl disable $1 2>/dev/null
     else
-        /sbin/chkconfig $1 off 2>/dev/null
+        /usr/sbin/update-rc.d $1 disable 2>/dev/null
     fi
 }
 
--- a/scripts/system_environment/enable_auto_updates
+++ b/scripts/system_environment/enable_auto_updates
@@ -15,20 +15,5 @@
 # Need to activate if new install or if they were previously running 3.3.2 or older
 if [[ "$1" == "new" || $MAJOR -lt 3 || ( $MAJOR -eq 3 && $MINOR -lt 4 ) ]]; then
     echo "Enabling Automatic Updates"
-    if type systemctl &>/dev/null; then
-        systemctl enable yum-cron
-    else
-        chkconfig --add yum-cron
-        chkconfig yum-cron on
-    fi
-fi
-
-# Handle updating configs here
-grep "<service yum_cron>" /etc/perfsonar/toolkit/configdaemon.conf > /dev/null 2>&1
-if [ $? != 0 ]; then
-    sed -i '/<access>/a \\t<service yum_cron> \
-		\t\trestart	\t\t 1 \
-		\t\tstart \t\t 1 \
-		\t\tstop \t\t 1 \
-	\t</service>' /etc/perfsonar/toolkit/configdaemon.conf
+    echo "-q -q dist-upgrade" > /etc/cron-apt/action.d/5-upgrade
 fi
--- a/scripts/system_environment/enable_nscd
+++ b/scripts/system_environment/enable_nscd
@@ -7,5 +7,5 @@
 if type systemctl &>/dev/null; then
     systemctl enable nscd
 else
-    /sbin/chkconfig nscd on
+    /usr/sbin/update-rc.d nscd enable
 fi
