--- a/init_scripts/perfsonar-configure_nic_parameters
+++ b/init_scripts/perfsonar-configure_nic_parameters
@@ -9,7 +9,7 @@
 ### END INIT INFO
 
 # Source function library.
-. /etc/init.d/functions
+. /lib/lsb/init-functions
 
 #always set these on perfsonar hosts
 DISABLE_TCP_OFFLOAD=1
@@ -18,14 +18,8 @@
 
 # find all the interfaces besides loopback.
 # ignore aliases, alternative configurations, and editor backup files
-interfaces=$(ls /etc/sysconfig/network-scripts/ifcfg* | \
-            LANG=C sed -e "$__sed_discard_ignored_files" \
-                       -e '/\(ifcfg-lo\|:\|ifcfg-.*-range\)/d' \
-                       -e '/ifcfg-[A-Za-z0-9\._-]\+$/ { s/^ifcfg-//g;s/[0-9]/ &/}' | \
-            LANG=C sort -k 1,1 -k 2n | \
-            LANG=C sed 's/ //' | \
-            LANG=C cut -d/ -f 5 | \
-        LANG=C sed -e 's/ifcfg-//g' )
+interfaces=$(/sbin/ifquery --all --list | grep -v -w lo)
+interfaces="$interfaces $(/sbin/ifquery --allow hotplug --list)"
 
 start() {
 ret=0
@@ -43,33 +37,33 @@
         RETVAL=$?
         if [ $RETVAL -eq 0 ]; then
             # apply multi-ethtool settings
-            echo -n $"disabling interrupt coalescing for interface $interface: "
+            log_action_begin_msg "Disabling interrupt coalescing for interface $interface"
             IC_OFF $interface
             if [ $? -eq 0 ]; then
-                success; echo
+                log_action_end_msg $?
             else
-                failure; echo; ret=1
+                log_action_end_msg $?; ret=1
             fi
         fi
     fi
 
     if [ $DISABLE_TCP_OFFLOAD ]; then
-        echo -n $"disabling TCP offload for interface $interface: "
+        log_action_begin_msg "Disabling TCP offload for interface $interface"
         TSO_OFF $interface
         if [ $? -eq 0 ]; then
-            success; echo
+            log_action_end_msg $?
         else
-            failure; echo; ret=1
+            log_action_end_msg $?; ret=1
         fi
     fi
 
     if [ $TXQUEUELEN -ne 0 ]; then
-        echo -n $"Setting txqueuelen for $interface to $TXQUEUELEN: "
+        log_action_begin_msg "Setting txqueuelen for $interface to $TXQUEUELEN"
         TXQUEUELEN_SET $interface $TXQUEUELEN
         if [ $? -eq 0 ]; then
-            success; echo
+            log_action_end_msg $?
         else
-            failure; echo; ret=1
+            log_action_end_msg $?; ret=1
         fi
     fi
 done
@@ -84,29 +78,28 @@
 IC_OFF() {
 IC_RET=0
 for i in "$1"; do
-    if ! /usr/sbin/ethtool -c $i | grep -q '^rx-frames: 1$'; then
-         /usr/sbin/ethtool -C $i rx-frames 1 || IC_RET=$?
+    if ! /sbin/ethtool -c $i | grep -q '^rx-frames: 1$'; then
+         /sbin/ethtool -C $i rx-frames 1 || IC_RET=$?
     fi
 
-    if ! /usr/sbin/ethtool -c $i | grep -q '^rx-usecs: 0$'; then
-         /usr/sbin/ethtool -C $i rx-usecs 0 || IC_RET=$?
+    if ! /sbin/ethtool -c $i | grep -q '^rx-usecs: 0$'; then
+         /sbin/ethtool -C $i rx-usecs 0 || IC_RET=$?
     fi
 done
 return $IC_RET
 }
 
 TSO_ON() {
-/usr/sbin/ethtool -K "$1" tso on
+/sbin/ethtool -K "$1" tso on
 }
 
 TSO_OFF() {
-/usr/sbin/ethtool -K "$1" tso off
+/sbin/ethtool -K "$1" tso off
 }
 
 
 case "$1" in
     start)
-        [ -f "$VAR_SUBSYS_TCP_TUNING" ] && exit 0
         start
         RETVAL=$?
         ;;
--- a/scripts/system_environment/disable_unwanted_services
+++ b/scripts/system_environment/disable_unwanted_services
@@ -6,9 +6,9 @@
 
 function disable_service() {
     if type systemctl &>/dev/null; then
-        systemctl disable $1 2>/dev/null
+        systemctl disable $1 &>/dev/null
     else
-        /sbin/chkconfig $1 off 2>/dev/null
+        /usr/sbin/update-rc.d $1 disable &>/dev/null
     fi
 }
 
--- a/scripts/system_environment/enable_auto_updates
+++ b/scripts/system_environment/enable_auto_updates
@@ -15,20 +15,8 @@
 # Need to activate if new install or if they were previously running 3.3.2 or older
 if [[ "$1" == "new" || $MAJOR -lt 3 || ( $MAJOR -eq 3 && $MINOR -lt 4 ) ]]; then
     echo "Enabling Automatic Updates"
-    if type systemctl &>/dev/null; then
-        systemctl enable yum-cron
-    else
-        chkconfig --add yum-cron
-        chkconfig yum-cron on
-    fi
-fi
-
-# Handle updating configs here
-grep "<service yum_cron>" /etc/perfsonar/toolkit/configdaemon.conf > /dev/null 2>&1
-if [ $? != 0 ]; then
-    sed -i '/<access>/a \\t<service yum_cron> \
-		\t\trestart	\t\t 1 \
-		\t\tstart \t\t 1 \
-		\t\tstop \t\t 1 \
-	\t</service>' /etc/perfsonar/toolkit/configdaemon.conf
+    echo 'APT::Periodic::Update-Package-Lists "1";' > /etc/apt/apt.conf.d/60unattended-upgrades-perfsonar
+    echo 'APT::Periodic::Unattended-Upgrade "1";' >> /etc/apt/apt.conf.d/60unattended-upgrades-perfsonar
+    echo 'APT::Periodic::AutocleanInterval "31";' >> /etc/apt/apt.conf.d/60unattended-upgrades-perfsonar
+    echo 'Unattended-Upgrade::Origins-Pattern:: "origin=perfSONAR";' >> /etc/apt/apt.conf.d/60unattended-upgrades-perfsonar
 fi
--- a/scripts/system_environment/enable_nscd
+++ b/scripts/system_environment/enable_nscd
@@ -7,5 +7,5 @@
 if type systemctl &>/dev/null; then
     systemctl enable nscd
 else
-    /sbin/chkconfig nscd on
+    /usr/sbin/update-rc.d nscd enable
 fi
