--- a/scripts/configure_firewall	2020-11-18 15:41:56.487041272 +0000
+++ b/scripts/configure_firewall	2020-11-18 15:41:57.611048059 +0000
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
