Index: toolkit/t/host-get_metadata.t
===================================================================
--- toolkit.orig/t/host-get_metadata.t	2018-12-18 21:23:20.676216000 +0000
+++ toolkit/t/host-get_metadata.t	2018-12-18 21:25:09.356705031 +0000
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
Index: toolkit/t/host-get_services.t
===================================================================
--- toolkit.orig/t/host-get_services.t	2018-12-18 21:23:20.676216000 +0000
+++ toolkit/t/host-get_services.t	2018-12-18 21:25:09.357296098 +0000
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
Index: toolkit/t/host-get_system_health.t
===================================================================
--- toolkit.orig/t/host-get_system_health.t	2018-12-18 21:23:20.676216000 +0000
+++ toolkit/t/host-get_system_health.t	2018-12-18 21:25:09.357716747 +0000
@@ -9,6 +9,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
Index: toolkit/t/host-update_metadata.t
===================================================================
--- toolkit.orig/t/host-update_metadata.t	2018-12-18 21:23:20.676216000 +0000
+++ toolkit/t/host-update_metadata.t	2018-12-18 21:25:09.358160291 +0000
@@ -13,6 +13,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
