--- a/scripts/configure_firewall	2020-02-19 15:37:03.724920596 +0000
+++ b/scripts/configure_firewall	2020-02-19 15:37:04.991938268 +0000
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
