Index: toolkit/scripts/configure_firewall
===================================================================
--- toolkit.orig/scripts/configure_firewall	2018-06-26 17:45:53.000000000 +0000
+++ toolkit/scripts/configure_firewall	2018-06-26 18:10:46.000000000 +0000
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
