--- a/scripts/system_environment/configure_ntpd
+++ b/scripts/system_environment/configure_ntpd
@@ -45,7 +45,7 @@
 
 file=/etc/ntp/step-tickers
 
-if [ ! -s $file ]; then
+if [ -e $file -a ! -s $file ]; then
 cat >$file <<EOF
 owamp.chic.net.internet2.edu 
 owamp.hous.net.internet2.edu
@@ -61,5 +61,5 @@
 # Auto-select NTP servers based on proximity
 ####################
 if [ "$1" == "new" ]; then
-    /opt/perfsonar_ps/toolkit/scripts/autoselect_ntp_servers
+    /usr/lib/perfsonar/scripts/autoselect_ntp_servers
 fi
--- a/scripts/system_environment/enable_ntpd
+++ b/scripts/system_environment/enable_ntpd
@@ -4,4 +4,4 @@
 # Enable NTP. Does not matter if this is running in an 'upgrade' or 'new
 # install' context.
 #######################
-/sbin/chkconfig ntpd on
+update-rc.d ntp defaults
