--- a/scripts/configure_firewall	2020-03-26 16:26:39.090621930 -0400
+++ b/scripts/configure_firewall	2020-03-26 16:27:03.823070959 -0400
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
