--- a/scripts/configure_firewall	2021-03-17 01:09:21.526599510 +0000
+++ b/scripts/configure_firewall	2021-03-17 01:09:22.525625172 +0000
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
