--- a/scripts/apache-toolkit_web_gui.conf
+++ b/scripts/apache-toolkit_web_gui.conf
@@ -4,6 +4,9 @@
 RewriteRule ^/toolkit-old/admin/(cacti.*) https://%{SERVER_NAME}/$1 [R,L]
 RewriteRule ^/toolkit-old/gui/(cacti.*) https://%{SERVER_NAME}/$1 [R,L]
 
+AddExternalAuth pwauth /usr/sbin/pwauth
+SetExternalAuthMethod pwauth pipe
+
 #Old GUI
 Alias /toolkit-old /usr/lib/perfsonar/web/root
 <Directory "/usr/lib/perfsonar/web/root">
@@ -23,9 +26,11 @@
     Order allow,deny
     Allow from all
 
-    AuthShadow on
     AuthType Basic
     AuthName "Password Required"
+    AuthBasicProvider external
+    AuthExternal pwauth
+    AuthzUnixgroup on
     Require group psadmin
 </Directory>
 
@@ -37,9 +42,11 @@
     Order allow,deny
     Allow from all
 
-    AuthShadow on
     AuthType Basic
     AuthName "Password Required"
+    AuthBasicProvider external
+    AuthExternal pwauth
+    AuthzUnixgroup on
     Require group psadmin
 </Directory>
 
@@ -117,9 +124,12 @@
     AddHandler cgi-script .cgi
     Order allow,deny
     Allow from all
-    AuthShadow on
+
     AuthType Basic
     AuthName "Password Required"
+    AuthBasicProvider external
+    AuthExternal pwauth
+    AuthzUnixgroup on
     Require group psadmin
 </Location>
 
@@ -137,9 +147,11 @@
     Order allow,deny
     Allow from all
 
-    AuthShadow on
     AuthType Basic
     AuthName "Password Required"
+    AuthBasicProvider external
+    AuthExternal pwauth
+    AuthzUnixgroup on
     Require group psadmin
 </Directory>
 
