--- a/scripts/configure_firewall	2019-08-09 13:06:16.578628632 +0000
+++ b/scripts/configure_firewall	2019-08-09 13:06:17.344637124 +0000
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
