--- a/scripts/system_environment/configure_esmond	2020-02-19 15:37:03.724920596 +0000
+++ b/scripts/system_environment/configure_esmond	2020-02-19 15:37:05.325942926 +0000
@@ -11,16 +11,12 @@
 fi
 
 #set esmond env variables
-export ESMOND_ROOT=/usr/lib/esmond
-export ESMOND_CONF=/etc/esmond/esmond.conf
+. /etc/default/esmond
+export ESMOND_ROOT ESMOND_CONF
 export DJANGO_SETTINGS_MODULE=esmond.settings
 
-#initialize python
-cd $ESMOND_ROOT
-. bin/activate
-
 #create api key
-KEY=`python esmond/manage.py add_api_key_user perfsonar 2> /dev/null | grep "Key:" | cut -f2 -d " "`
+KEY=`python /usr/lib/python2.7/dist-packages/esmond/manage.py add_api_key_user perfsonar 2> /dev/null | grep "Key:" | cut -f2 -d " "`
 
 #setup localhost measurement archive
 if [ -n "$KEY" ]; then
--- a/scripts/system_environment/configure_fail2ban	2020-02-19 15:37:03.724920596 +0000
+++ b/scripts/system_environment/configure_fail2ban	2020-02-19 15:37:05.325942926 +0000
@@ -10,5 +10,5 @@
     echo "" >> /etc/fail2ban/jail.local
     echo "[sshd]" >> /etc/fail2ban/jail.local
     echo "enabled = true" >> /etc/fail2ban/jail.local
-    /sbin/service fail2ban restart
+    service fail2ban restart
 fi
--- a/scripts/system_environment/upgrade_configdaemon	2020-02-19 15:37:03.726920624 +0000
+++ b/scripts/system_environment/upgrade_configdaemon	2020-02-19 15:37:05.325942926 +0000
@@ -87,5 +87,5 @@
 EOF
 
 #centos 7 should not need this
-/sbin/service perfsonar-configdaemon restart
+service perfsonar-configdaemon restart
 fi
--- a/scripts/system_environment/upgrade_fix_permissions	2020-02-19 15:37:03.726920624 +0000
+++ b/scripts/system_environment/upgrade_fix_permissions	2020-02-19 15:37:05.325942926 +0000
@@ -8,10 +8,10 @@
 chown -R perfsonar:perfsonar /var/log/perfsonar
 
 mkdir -p /var/log/cacti
-chown -R apache /var/log/cacti
+chown -R www-data /var/log/cacti
 
 mkdir -p /var/log/perfsonar/web_admin
-chown -R apache:perfsonar /var/log/perfsonar/web_admin
+chown -R www-data:perfsonar /var/log/perfsonar/web_admin
 
 # Make sure that the various /var/lib/perfsonar directories are correct.
 mkdir -p /var/lib/perfsonar
@@ -22,7 +22,7 @@
 
 # Toolkit odds and ends
 mkdir -p /var/run/web_admin_sessions
-chown -R apache /var/run/web_admin_sessions
+chown -R www-data /var/run/web_admin_sessions
 
 #Try cacti data, but don't complain if it's not there
-chown -R apache /var/lib/cacti/rra 2> /dev/null
+chown -R www-data /var/lib/cacti/rra 2> /dev/null
--- a/scripts/configure_apache_security	2020-02-19 15:37:03.723920582 +0000
+++ b/scripts/configure_apache_security	2020-02-19 15:37:05.325942926 +0000
@@ -5,14 +5,18 @@
 # updates. Settings will be in apache-security.conf but if someone wants 
 # to override those settings then they can do so in ssl.comnf after first install
 #######################
-if [ "$1" == "new" ]; then
-    sed -i 's|^SSLProtocol\(.*\)|#SSLProtocol\1|g' /etc/httpd/conf.d/ssl.conf
-    sed -i 's|^SSLCipherSuite\(.*\)|#SSLCipherSuite\1|g'  /etc/httpd/conf.d/ssl.conf
-fi
+for CONF in /etc/apache2/sites-available/{default-ssl,default-ssl.conf}; do
+    if [ -f $CONF ]; then
+        sed -i 's|^\([[:space:]]*\)SSLProtocol|\1#SSLProtocol|g' $CONF
+        sed -i 's|^\([[:space:]]*\)SSLCipherSuite|\1#SSLCipherSuite|g' $CONF
+    fi
+done
 
 #######################
 # Get rid of default welcome page
 #######################
-if [ -f "/etc/httpd/conf.d/welcome.conf" ]; then
-    mv -f /etc/httpd/conf.d/welcome.conf /etc/httpd/conf.d/welcome.conf.bak
-fi
\ No newline at end of file
+for CONF in /etc/apache2/sites-available/{default,default-ssl,000-default.conf,default-ssl.conf}; do
+    if [ -f $CONF ]; then
+       sed -i 's/DocumentRoot \/var\/www\(\/html\)\?$/DocumentRoot \/var\/www\/perfsonar/' $CONF
+    fi
+done
--- a/scripts/system_environment/testpoint/configure_syslog_local5_location	2020-02-19 15:37:03.726920624 +0000
+++ b/scripts/system_environment/testpoint/configure_syslog_local5_location	2020-02-19 15:37:05.325942926 +0000
@@ -11,32 +11,17 @@
     mv /var/log/perfsonar/owamp.log /var/log/perfsonar/owamp.log.old  
 fi
 
+# Rename if using pre-4.2.3 log
+rm -f /etc/rsyslog.d/owamp-syslog.conf
+rm -f /etc/logrotate.d/perfsonar-toolkit
+
 # Set 'local5' output location to /var/log/perfsonar/owamp.log
-grep ^local5 /etc/rsyslog.d/owamp-syslog.conf &> /dev/null
+grep ^local5 /etc/rsyslog.d/30-owamp-syslog.conf &> /dev/null
 if [ $? != 0 ]; then
-cat >> /etc/rsyslog.d/owamp-syslog.conf <<EOF                                          
-# Save bwctl and owamp messages to /var/log/perfsonar/owamp.log
+cat >> /etc/rsyslog.d/30-owamp-syslog.conf <<EOF
+# Save owamp and twamp messages to /var/log/perfsonar/owamp.log
 local5.*                                                -/var/log/perfsonar/owamp.log
-EOF
-fi
-
-# Disable sync on /var/log/messages to save IO
-sed 's/ \/var\/log\/messages/ -\/var\/log\/messages/g' /etc/rsyslog.conf > /etc/rsyslog.conf.tmp
-# Cleanup any incorrect --/var/log/messages entries from earlier versions
-sed -i 's/--*\/var\/log\/messages/-\/var\/log\/messages/g' /etc/rsyslog.conf.tmp
-mv /etc/rsyslog.conf.tmp /etc/rsyslog.conf
-
-# Make sure that the owamp/bwctl log file gets rotated regularly
-grep owamp.log /etc/logrotate.d/perfsonar-toolkit &> /dev/null
-if [ $? != 0 ]; then
-cat >>/etc/logrotate.d/perfsonar-toolkit <<EOF
-/var/log/perfsonar/owamp.log {
-    sharedscripts
-    postrotate
-        /bin/kill -HUP \`cat /var/run/syslogd.pid 2> /dev/null\` 2> /dev/null || true
-        /bin/kill -HUP \`cat /var/run/rsyslogd.pid 2> /dev/null\` 2> /dev/null || true
-    endscript
-}
+& stop
 EOF
 fi
 
@@ -49,4 +34,5 @@
 # The log file needs to be created initially so that the log file isn't the
 # default of root:root and only readable by root.
 touch /var/log/perfsonar/owamp.log
-chown perfsonar:perfsonar /var/log/perfsonar/owamp.log
+chmod 664 /var/log/perfsonar/owamp.log
+chown perfsonar:adm /var/log/perfsonar/owamp.log
