--- a/scripts/configure_firewall	2019-07-18 21:26:06.031899000 +0000
+++ b/scripts/configure_firewall	2019-07-18 21:33:19.591139000 +0000
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
