Index: toolkit/scripts/configure_firewall
===================================================================
--- toolkit.orig/scripts/configure_firewall	2017-11-07 12:39:38.000000000 +0000
+++ toolkit/scripts/configure_firewall	2017-12-19 16:29:07.000000000 +0000
@@ -35,7 +35,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
