--- a/scripts/apache-toolkit_web_gui.conf	2021-05-18 13:01:06.958982333 +0000
+++ b/scripts/apache-toolkit_web_gui.conf	2021-05-18 13:01:08.059996212 +0000
@@ -7,6 +7,11 @@
 # New GUI
 Alias /toolkit /usr/lib/perfsonar/web-ng/root
 
+# Redirects requests to "/" to "/toolkit". It's done in this strange way to
+# avoid confusing people who enter an IP address and would get redirected to
+# the hostname, or vice versa.
+RedirectMatch ^/$ /toolkit/
+
 # Redirect from /index.cgi?format=json to the new host summary page
 RewriteCond %{QUERY_STRING} format=json
 RewriteRule ^/toolkit/index.cgi /toolkit/services/host.cgi?method=get_summary [R,L]
@@ -46,10 +51,8 @@
 # But, allow anonymous access via port 80 (http)
 # NOTE: Need to set non-https re-write rules and redirects here
 <VirtualHost *:80>
-    # Redirects requests to "/" to "/toolkit". It's done in this strange way to
-    # avoid confusing people who enter an IP address and would get redirected to
-    # the hostname, or vice versa.
-    RedirectMatch ^/$ /toolkit/
+    # Hide default index page
+    DocumentRoot /var/www/perfsonar
 
     # Redirect from /index.cgi?format=json to the new host summary page
     RewriteCond %{QUERY_STRING} format=json
@@ -98,23 +101,11 @@
     Options +FollowSymLinks -MultiViews +ExecCGI
     DirectoryIndex index.cgi index.php index.html
     AddHandler cgi-script .cgi
-    <IfVersion < 2.4>
-      Order allow,deny
-      Allow from all
-    </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Location>
 
 # Allow indexes on logs
@@ -128,23 +119,11 @@
     DirectoryIndex index.cgi index.php index.html
     AddHandler cgi-script .cgi
     AllowOverride All
-    <IfVersion < 2.4>
-      Order allow,deny
-      Allow from all
-    </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Directory>
 
 ##
--- a/scripts/add_pssudo_user	2021-05-18 13:01:06.958982333 +0000
+++ b/scripts/add_pssudo_user	2021-05-18 13:01:08.059996212 +0000
@@ -104,7 +104,7 @@
     fout.write(buffer)
 
 #restart ssh
-retVal = os.system("/sbin/service sshd restart")
+retVal = os.system("service ssh restart")
 if (retVal != 0):
     print(Internet2Consts.YELLOW + "Error: couldn't restart sshd. Root SSH will be disabled on next restart." + Internet2Consts.NORMAL)
     sys.exit(1)
--- a/scripts/clean_esmond_db.sh	2021-05-18 13:01:06.958982333 +0000
+++ b/scripts/clean_esmond_db.sh	2021-05-18 13:01:08.059996212 +0000
@@ -9,14 +9,14 @@
 fi
 
 #run ps_remove_data.py to delete expired data
-export ESMOND_ROOT=/usr/lib/esmond
-export ESMOND_CONF=/etc/esmond/esmond.conf
+
+. /etc/default/esmond
+export ESMOND_ROOT
+export ESMOND_CONF
 export DJANGO_SETTINGS_MODULE=esmond.settings
-cd $ESMOND_ROOT
-. bin/activate
-python ./util/ps_remove_data.py -c /etc/perfsonar/toolkit/clean_esmond_db.conf
+python3 /usr/share/esmond/util/ps_remove_data.py -c /etc/perfsonar/toolkit/clean_esmond_db.conf
 
 #force a compaction
 if [ "$ESMOND_CLEANER_MULTI_NODE" != "1" ]; then
     nodetool compact esmond
-fi
\ No newline at end of file
+fi
--- a/init_scripts/perfsonar-configure_nic_parameters	2021-05-18 13:01:06.952982258 +0000
+++ b/init_scripts/perfsonar-configure_nic_parameters	2021-05-18 13:01:08.059996212 +0000
@@ -1,17 +1,12 @@
 #!/bin/sh
-#
-# $Id$
-#
-# multi-ethtool
-#
-# chkconfig: - 95 99
-# description:	pS-Performance Toolkit script to configure the NIC parameters
-# 
-# see:
-# http://code.google.com/p/perfsonar-ps/issues/detail?id=122
-# https://bugzilla.redhat.com/show_bug.cgi?id=179877
-#
-#
+### BEGIN INIT INFO
+# Provides:          perfsonar-configure_nic_parameters
+# Required-Start:    $local_fs $remote_fs $network
+# Required-Stop:     $local_fs $remote_fs $network
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: pS-Performance Toolkit script to configure the NIC parameters
+### END INIT INFO
 
 # Source function library.
 . /etc/init.d/functions
@@ -140,6 +135,10 @@
         start
         RETVAL=$?
         ;;
+    stop)
+        # to make systemd happy
+        RETVAL=0
+        ;;
     *)
         echo $"Usage: $0 {start}"
         RETVAL=2
--- a/init_scripts/perfsonar-generate_motd	2021-05-18 13:01:06.952982258 +0000
+++ b/init_scripts/perfsonar-generate_motd	2021-05-18 13:01:08.059996212 +0000
@@ -1,11 +1,15 @@
 #!/usr/bin/perl
-#
-# pS-Performance Toolkit script that changes the MOTD to include the toolkit
-# version number as well as the URL to go to configure the toolkit.
-#
-# chkconfig: 2345 99 99
-# description: pS-Performance Toolkit MOTD maintenance script
-#
+### BEGIN INIT INFO
+# Provides:          perfsonar-generate_motd
+# Required-Start:    $local_fs $remote_fs
+# Required-Stop:     $local_fs $remote_fs
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: pS-Performance Toolkit MOTD maintenance script
+# Description:       pS-Performance Toolkit script that changes the MOTD
+#                    to include the toolkit version number as well as
+#                    the URL to go to configure the toolkit.
+### END INIT INFO
 
 use strict;
 use warnings;
--- a/web-ng/root/gui/reverse_traceroute.cgi	2021-05-18 13:01:06.967982446 +0000
+++ b/web-ng/root/gui/reverse_traceroute.cgi	2021-05-18 13:01:08.062996249 +0000
@@ -273,7 +273,7 @@
 #########################################################################
 # Build the executable function for tracepath
 elsif($function eq "tracepath") {
-  $Tr="/bin/tracepath";
+  $Tr="tracepath";
   @Tropts="";
 }
 #########################################################################
--- a/etc/perfsonar_ulimit.conf	2021-05-18 13:01:06.951982245 +0000
+++ b/etc/perfsonar_ulimit.conf	2021-05-18 13:01:08.062996249 +0000
@@ -1,8 +1,8 @@
 # Increases process limit and number of files open for various perfsonar tools
 
 pscheduler  soft    nproc     4096
-apache      soft    nproc     4096
+www-data    soft    nproc     4096
 pscheduler  soft    nofile    4096
-apache      soft    nofile    4096
+www-data    soft    nofile    4096
 perfsonar   soft    nofile    4096
-owamp       soft    nofile    4096
\ No newline at end of file
+owamp       soft    nofile    4096
