--- a/scripts/configure_firewall	2022-01-12 09:33:16.424743927 -0500
+++ b/scripts/configure_firewall	2022-01-12 09:33:18.427173892 -0500
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
