--- toolkit.orig/t/host-get_metadata.t	2018-07-17 13:45:35.264999366 +0200
+++ toolkit/t/host-get_metadata.t	2018-07-17 13:45:35.252999198 +0200
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
--- toolkit.orig/t/host-get_services.t	2018-07-17 13:45:35.264999366 +0200
+++ toolkit/t/host-get_services.t	2018-07-17 13:45:54.437268386 +0200
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
--- toolkit.orig/t/host-get_system_health.t	2018-07-17 13:45:35.264999366 +0200
+++ toolkit/t/host-get_system_health.t	2018-07-17 13:45:35.252999198 +0200
@@ -9,6 +9,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
--- toolkit.orig/t/host-update_metadata.t	2018-07-17 13:45:35.264999366 +0200
+++ toolkit/t/host-update_metadata.t	2018-07-17 13:45:35.252999198 +0200
@@ -13,6 +13,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
