--- a/scripts/apache-toolkit_web_gui.conf
+++ b/scripts/apache-toolkit_web_gui.conf
@@ -88,18 +88,10 @@
       Allow from all
     </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Location>
 
 # Allow indexes on logs
@@ -118,18 +110,10 @@
       Allow from all
     </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Directory>
 
 ##
