--- toolkit.orig/t/host-get_metadata.t	2018-07-17 12:31:45.606922916 +0200
+++ toolkit/t/host-get_metadata.t	2018-07-17 12:31:45.590922692 +0200
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
--- toolkit.orig/t/host-get_services.t	2018-07-17 12:31:45.606922916 +0200
+++ toolkit/t/host-get_services.t	2018-07-17 12:31:52.275016277 +0200
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
@@ -138,6 +139,13 @@
                 'enabled' => $enabled
             },
             {
+                'is_running' => $running,
+                'addresses' => [],
+                'version' => $version,
+                'name' => 'twamp',
+                'enabled' => $enabled
+            },
+            {
                 'is_running' => $running,
                 'addresses' => $pscheduler_addresses,
                 'version' => $version,
--- toolkit.orig/t/host-get_system_health.t	2018-07-17 12:31:45.606922916 +0200
+++ toolkit/t/host-get_system_health.t	2018-07-17 12:31:45.594922748 +0200
@@ -9,6 +9,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
--- toolkit.orig/t/host-update_metadata.t	2018-07-17 12:31:45.606922916 +0200
+++ toolkit/t/host-update_metadata.t	2018-07-17 12:31:45.594922748 +0200
@@ -13,6 +13,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
