Index: toolkit/scripts/apache-toolkit_web_gui.conf
===================================================================
--- toolkit.orig/scripts/apache-toolkit_web_gui.conf	2017-12-19 16:30:46.000000000 +0000
+++ toolkit/scripts/apache-toolkit_web_gui.conf	2017-12-19 16:41:08.000000000 +0000
@@ -46,6 +46,9 @@
 # But, allow anonymous access via port 80 (http)
 # NOTE: Need to set non-https re-write rules and redirects here
 <VirtualHost *:80>
+    # Hide default index page
+    DocumentRoot /var/www/perfsonar
+
     # Redirects requests to "/" to "/toolkit". It's done in this strange way to
     # avoid confusing people who enter an IP address and would get redirected to
     # the hostname, or vice versa.
@@ -98,23 +101,11 @@
     Options +FollowSymLinks -MultiViews +ExecCGI
     DirectoryIndex index.cgi index.php index.html
     AddHandler cgi-script .cgi
-    <IfVersion < 2.4>
-      Order allow,deny
-      Allow from all
-    </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Location>
 
 # Allow indexes on logs
@@ -128,23 +119,11 @@
     DirectoryIndex index.cgi index.php index.html
     AddHandler cgi-script .cgi
     AllowOverride All
-    <IfVersion < 2.4>
-      Order allow,deny
-      Allow from all
-    </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Directory>
 
 ##
Index: toolkit/scripts/add_pssudo_user
===================================================================
--- toolkit.orig/scripts/add_pssudo_user	2017-12-19 16:30:46.000000000 +0000
+++ toolkit/scripts/add_pssudo_user	2017-12-19 16:41:08.000000000 +0000
@@ -59,7 +59,7 @@
     sys.exit(1)
 
 user = ans
-cmd = "/usr/sbin/adduser -m -G pssudo"
+cmd = "/usr/sbin/useradd -m -G pssudo"
 cmd += " " + user
 
 retVal = os.system(cmd)
@@ -95,7 +95,7 @@
     fout.write(buffer)
 
 #restart ssh
-retVal = os.system("/sbin/service sshd restart")
+retVal = os.system("service ssh restart")
 if (retVal != 0):
     print Internet2Consts.YELLOW + "Error: couldn't restart sshd. Root SSH will be disabled on next restart." + Internet2Consts.NORMAL
     sys.exit(1)
Index: toolkit/scripts/clean_esmond_db.sh
===================================================================
--- toolkit.orig/scripts/clean_esmond_db.sh	2017-12-19 16:30:46.000000000 +0000
+++ toolkit/scripts/clean_esmond_db.sh	2017-12-19 16:41:08.000000000 +0000
@@ -9,15 +9,13 @@
 fi
 
 #run ps_remove_data.py to delete expired data
-export ESMOND_ROOT=/usr/lib/esmond
-export ESMOND_CONF=/etc/esmond/esmond.conf
+
+. /etc/default/esmond
+export ESMOND_ROOT
+export ESMOND_CONF
 export DJANGO_SETTINGS_MODULE=esmond.settings
-cd $ESMOND_ROOT
-if [ -e /opt/rh/python27/enable ]; then
-    source /opt/rh/python27/enable
-fi
-. bin/activate
-python ./util/ps_remove_data.py -c /etc/perfsonar/toolkit/clean_esmond_db.conf
+
+python /usr/share/esmond/util/ps_remove_data.py -c /etc/perfsonar/toolkit/clean_esmond_db.conf
 
 #force a compaction
 if [ "$ESMOND_CLEANER_MULTI_NODE" != "1" ]; then
Index: toolkit/init_scripts/perfsonar-configure_nic_parameters
===================================================================
--- toolkit.orig/init_scripts/perfsonar-configure_nic_parameters	2017-12-19 16:30:46.000000000 +0000
+++ toolkit/init_scripts/perfsonar-configure_nic_parameters	2017-12-19 16:41:08.000000000 +0000
@@ -1,17 +1,12 @@
 #!/bin/sh
-#
-# $Id$
-#
-# multi-ethtool
-#
-# chkconfig: - 95 99
-# description:	pS-Performance Toolkit script to configure the NIC parameters
-# 
-# see:
-# http://code.google.com/p/perfsonar-ps/issues/detail?id=122
-# https://bugzilla.redhat.com/show_bug.cgi?id=179877
-#
-#
+### BEGIN INIT INFO
+# Provides:          perfsonar-configure_nic_parameters
+# Required-Start:    $local_fs $remote_fs $network
+# Required-Stop:     $local_fs $remote_fs $network
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: pS-Performance Toolkit script to configure the NIC parameters
+### END INIT INFO
 
 # Source function library.
 . /etc/init.d/functions
@@ -115,6 +110,10 @@
         start
         RETVAL=$?
         ;;
+    stop)
+        # to make systemd happy
+        RETVAL=0
+        ;;
     *)
         echo $"Usage: $0 {start}"
         RETVAL=2
Index: toolkit/init_scripts/perfsonar-generate_motd
===================================================================
--- toolkit.orig/init_scripts/perfsonar-generate_motd	2017-12-19 16:30:46.000000000 +0000
+++ toolkit/init_scripts/perfsonar-generate_motd	2017-12-19 16:41:08.000000000 +0000
@@ -1,11 +1,15 @@
 #!/usr/bin/perl
-#
-# pS-Performance Toolkit script that changes the MOTD to include the toolkit
-# version number as well as the URL to go to configure the toolkit.
-#
-# chkconfig: 2345 99 99
-# description: pS-Performance Toolkit MOTD maintenance script
-#
+### BEGIN INIT INFO
+# Provides:          perfsonar-generate_motd
+# Required-Start:    $local_fs $remote_fs
+# Required-Stop:     $local_fs $remote_fs
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: pS-Performance Toolkit MOTD maintenance script
+# Description:       pS-Performance Toolkit script that changes the MOTD
+#                    to include the toolkit version number as well as
+#                    the URL to go to configure the toolkit.
+### END INIT INFO
 
 use strict;
 use warnings;
Index: toolkit/init_scripts/perfsonar-psb_to_esmond
===================================================================
--- toolkit.orig/init_scripts/perfsonar-psb_to_esmond	2017-12-19 16:30:46.000000000 +0000
+++ toolkit/init_scripts/perfsonar-psb_to_esmond	2017-12-19 16:41:08.000000000 +0000
@@ -1,10 +1,13 @@
 #!/bin/bash
-#
-# Init file for perfSONAR MA data conversion tool
-#
-# chkconfig: 2345 99 20
-# description: perfSONAR MA data conversion tool
-#
+### BEGIN INIT INFO
+# Provides:          perfsonar-psb_to_esmond
+# Required-Start:    $local_fs $remote_fs $network apache2 cassandra
+# Required-Stop:     $local_fs $remote_fs $network apache2 cassandra
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: perfSONAR MA data conversion tool
+# Description:       Init file for perfSONAR MA data conversion tool
+### END INIT INFO
 
 PREFIX=/usr/lib/perfsonar
 BINDIR=${PREFIX}/scripts
Index: toolkit/web-ng/root/gui/reverse_traceroute.cgi
===================================================================
--- toolkit.orig/web-ng/root/gui/reverse_traceroute.cgi	2017-12-19 16:30:46.000000000 +0000
+++ toolkit/web-ng/root/gui/reverse_traceroute.cgi	2017-12-19 16:41:08.000000000 +0000
@@ -255,7 +255,7 @@
 #########################################################################
 # Build the executable function for tracepath
 elsif($function eq "tracepath") {
-  $Tr="/bin/tracepath";
+  $Tr="tracepath";
   @Tropts=();
 }
 #########################################################################
