--- a/scripts/configure_firewall	2020-11-02 22:02:58.219950000 +0000
+++ b/scripts/configure_firewall	2020-11-02 22:03:49.178948401 +0000
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
