Index: toolkit/scripts/configure_firewall
===================================================================
--- toolkit.orig/scripts/configure_firewall	2018-09-13 07:42:00.397579820 +0200
+++ toolkit/scripts/configure_firewall	2018-09-13 08:03:19.451671564 +0200
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
