Index: toolkit/scripts/configure_ntpd
===================================================================
--- toolkit.orig/scripts/configure_ntpd	2018-03-03 17:17:46.000000000 +0000
+++ toolkit/scripts/configure_ntpd	2018-03-03 17:19:08.000000000 +0000
@@ -45,7 +45,7 @@
 
 file=/etc/ntp/step-tickers
 
-if [ ! -s $file ]; then
+if [ -e $file -a ! -s $file ]; then
 cat >$file <<EOF
 owamp.chic.net.internet2.edu 
 owamp.hous.net.internet2.edu
