--- a/scripts/nptoolkit-configure.py	2021-07-07 15:19:03.773236204 +0000
+++ b/scripts/nptoolkit-configure.py	2021-07-07 15:19:05.373272615 +0000
@@ -99,7 +99,7 @@
                     },
                     {
                          "description": "Change Timezone",
-                         "command": "/usr/bin/tzselect",
+                         "command": "/usr/sbin/dpkg-reconfigure tzdata",
                          "isConfiguredFunction": None,
                          "required": False,
                          "requiresReboot": True,
@@ -130,7 +130,7 @@
 
     for item in known_items:
         # Skip the item if it's associated command isn't available.
-        if (not os.path.exists(item["command"])):
+        if (not os.path.exists(item["command"].split(" ")[0])):
             continue
 
         item = toolkitItem(item["description"], item["command"], item["isConfiguredFunction"], item["required"], item["requiresReboot"])
--- a/scripts/install-optional-packages.py	2021-07-07 15:19:03.773236204 +0000
+++ b/scripts/install-optional-packages.py	2021-07-07 15:19:05.373272615 +0000
@@ -18,7 +18,7 @@
 
 sys.path.append(libdir)
 
-import yum
+import apt
 import Internet2Lib
 import Internet2Consts
 
@@ -30,7 +30,7 @@
     'perfsonar-toolkit-systemenv-testpoint'
 ]
 
-def installPackages(yumHandle, packageNames):
+def installPackages(apt_cache, packageNames):
     userChoice=[]
     for package in packageNames:
         print("\nWould you like to install "+package+"?")
@@ -40,37 +40,39 @@
     
     for package in userChoice:
         print("Installing "+package)
-        try:
-            yumHandle.install(name=package)
-        except yum.Errors.InstallError as err:
-            print("Error installing package"+str(err))
+        apt_cache[package].mark_install()
 
     if userChoice:
-        yumHandle.resolveDeps()
-        yumHandle.processTransaction()
+        try:
+            apt_cache.commit()
+        except Exception, err:
+            print ("Error installing package: "+str(err))
+
         print("\n Installed the following packages: \n")
         for package in userChoice:
             print(package)
 
-def findInstalledPackages(yumHandle, packageNames):
+def findInstalledPackages(apt_cache, packageNames):
     toInstall = []
     for package in packageNames:
-        if not yumHandle.rpmdb.searchNevra(name=package):
+        if not apt_cache[package].is_installed:
             toInstall.append(package)
             print(package)
     return toInstall
     
 ###MAIN 
-yHandle = yum.YumBase();
 if not Internet2Lib.isRoot():
     print(Internet2Consts.YELLOW + "You must run the perfSONAR install-optional-packages script as root." + Internet2Consts.NORMAL)
     sys.exit(1)
 
+apt_cache = apt.cache.Cache()
+apt_cache.update()
+
 #find what packages need to be installed
-packagesToInstall = findInstalledPackages(yHandle, optionalPackages)
+packagesToInstall = findInstalledPackages(apt_cache, optionalPackages)
 
 if packagesToInstall:
-    installPackages(yHandle, packagesToInstall)
+    installPackages(apt_cache, packagesToInstall)
 else:
     print("All optional packages have been installed \n")
     for package in optionalPackages:
