--- toolkit.orig/t/host-get_metadata.t	2017-06-20 14:56:26.000000000 +0200
+++ toolkit/t/host-get_metadata.t	2018-04-10 16:08:48.569781513 +0200
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
--- toolkit.orig/t/host-get_services.t	2018-04-10 14:29:08.985652175 +0200
+++ toolkit/t/host-get_services.t	2018-04-10 16:08:52.969778522 +0200
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
--- toolkit.orig/t/host-get_system_health.t	2017-06-20 14:56:26.000000000 +0200
+++ toolkit/t/host-get_system_health.t	2018-04-10 16:08:56.553776072 +0200
@@ -9,6 +9,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
--- toolkit.orig/t/host-update_metadata.t	2017-08-28 13:00:57.000000000 +0200
+++ toolkit/t/host-update_metadata.t	2018-04-10 16:09:01.017773032 +0200
@@ -13,6 +13,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
