Index: toolkit/t/host-get_metadata.t
===================================================================
--- toolkit.orig/t/host-get_metadata.t	2018-08-13 18:20:23.000000000 +0000
+++ toolkit/t/host-get_metadata.t	2018-08-24 17:31:53.000000000 +0000
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
Index: toolkit/t/host-get_services.t
===================================================================
--- toolkit.orig/t/host-get_services.t	2018-08-13 18:20:23.000000000 +0000
+++ toolkit/t/host-get_services.t	2018-08-24 17:31:53.000000000 +0000
@@ -6,6 +6,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
Index: toolkit/t/host-get_system_health.t
===================================================================
--- toolkit.orig/t/host-get_system_health.t	2018-08-13 18:20:23.000000000 +0000
+++ toolkit/t/host-get_system_health.t	2018-08-24 17:31:53.000000000 +0000
@@ -9,6 +9,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
Index: toolkit/t/host-update_metadata.t
===================================================================
--- toolkit.orig/t/host-update_metadata.t	2018-08-13 18:20:23.000000000 +0000
+++ toolkit/t/host-update_metadata.t	2018-08-24 17:31:53.000000000 +0000
@@ -13,6 +13,7 @@
 use FindBin qw($Bin);
 use lib "$Bin/lib";
 use lib "$Bin/../lib";
+use lib "/usr/lib/perfsonar/lib";
 use Log::Log4perl qw(:easy);
 Log::Log4perl->easy_init( {level => 'OFF'} );
 
