--- a/scripts/configure_firewall	2020-09-29 18:02:44.789556794 +0000
+++ b/scripts/configure_firewall	2020-09-29 18:02:45.957476176 +0000
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
