Index: toolkit/scripts/configure_ntpd
===================================================================
--- toolkit.orig/scripts/configure_ntpd	2018-05-25 16:32:26.000000000 +0000
+++ toolkit/scripts/configure_ntpd	2018-05-25 16:34:29.000000000 +0000
@@ -45,7 +45,7 @@
 
 file=/etc/ntp/step-tickers
 
-if [ ! -s $file ]; then
+if [ -e $file -a ! -s $file ]; then
 cat >$file <<EOF
 owamp.chic.net.internet2.edu 
 owamp.hous.net.internet2.edu
