--- a/scripts/configure_firewall	2022-10-27 10:33:48.411433692 -0400
+++ b/scripts/configure_firewall	2022-10-27 10:33:50.802792722 -0400
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
