Index: b/bin/config_daemon.pl
===================================================================
--- a/bin/config_daemon.pl	2024-05-09 23:38:59.145764000 +0200
+++ b/bin/config_daemon.pl	2024-05-10 00:28:53.658750273 +0200
@@ -29,7 +29,7 @@
 use Log::Log4perl qw/:easy/;
 
 # set the process name
-$0 = "toolkit_config_daemon.pl";
+$0 = "configdaemon.pl";
 
 my @child_pids = ();
 
@@ -70,7 +70,7 @@
 }
 
 if ( not $PIDFILE ) {
-    $PIDFILE = "/var/run/toolkit_config_daemon.pid";
+    $PIDFILE = "/var/run/configdaemon.pid";
 }
 
 ( $status, $res ) = lockPIDFile( $PIDFILE );
Index: b/etc/configdaemon.conf
===================================================================
--- a/etc/configdaemon.conf	2024-05-09 23:38:59.145917000 +0200
+++ b/etc/configdaemon.conf	2024-05-10 00:28:53.659479742 +0200
@@ -1,6 +1,6 @@
 address     127.0.0.1
 port        9000
-firewall_script /usr/lib/perfsonar/scripts/system_environment/configure_firewall
+firewall_script /usr/lib/perfsonar/scripts/configure_firewall
 
 <access>
     <service yum_cron>
