--- a/scripts/configure_firewall	2023-03-22 15:04:56.482761000 +0000
+++ b/scripts/configure_firewall	2023-03-31 13:24:40.288948659 +0000
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
