Index: b/scripts/system_environment/configure_fail2ban
===================================================================
--- a/scripts/system_environment/configure_fail2ban	2024-06-28 10:51:34.666480895 -0400
+++ b/scripts/system_environment/configure_fail2ban	2024-06-28 10:51:36.832832184 -0400
@@ -10,5 +10,5 @@
     echo "" >> /etc/fail2ban/jail.local
     echo "[sshd]" >> /etc/fail2ban/jail.local
     echo "enabled = true" >> /etc/fail2ban/jail.local
-    /sbin/service fail2ban restart
+    service fail2ban restart
 fi
Index: b/scripts/system_environment/upgrade_configdaemon
===================================================================
--- a/scripts/system_environment/upgrade_configdaemon	2024-06-28 10:51:34.667498760 -0400
+++ b/scripts/system_environment/upgrade_configdaemon	2024-06-28 10:51:36.833301346 -0400
@@ -87,5 +87,5 @@
 EOF
 
 #centos 7 should not need this
-/sbin/service perfsonar-configdaemon restart
+service perfsonar-configdaemon restart
 fi
Index: b/scripts/system_environment/upgrade_fix_permissions
===================================================================
--- a/scripts/system_environment/upgrade_fix_permissions	2024-06-28 10:51:34.667572093 -0400
+++ b/scripts/system_environment/upgrade_fix_permissions	2024-06-28 10:51:36.833587969 -0400
@@ -8,10 +8,10 @@
 chown -R perfsonar:perfsonar /var/log/perfsonar
 
 mkdir -p /var/log/cacti
-chown -R apache /var/log/cacti
+chown -R www-data /var/log/cacti
 
 mkdir -p /var/log/perfsonar/web_admin
-chown -R apache:perfsonar /var/log/perfsonar/web_admin
+chown -R www-data:perfsonar /var/log/perfsonar/web_admin
 
 # Make sure that the various /var/lib/perfsonar directories are correct.
 mkdir -p /var/lib/perfsonar
@@ -22,7 +22,7 @@
 
 # Toolkit odds and ends
 mkdir -p /var/run/web_admin_sessions
-chown -R apache /var/run/web_admin_sessions
+chown -R www-data /var/run/web_admin_sessions
 
 #Try cacti data, but don't complain if it's not there
-chown -R apache /var/lib/cacti/rra 2> /dev/null
+chown -R www-data /var/lib/cacti/rra 2> /dev/null
Index: b/scripts/system_environment/testpoint/configure_syslog_local5_location
===================================================================
--- a/scripts/system_environment/testpoint/configure_syslog_local5_location	2024-06-28 10:51:34.667132680 -0400
+++ b/scripts/system_environment/testpoint/configure_syslog_local5_location	2024-06-28 10:51:36.833893257 -0400
@@ -11,32 +11,17 @@
     mv /var/log/perfsonar/owamp.log /var/log/perfsonar/owamp.log.old  
 fi
 
+# Rename if using pre-4.2.3 log
+rm -f /etc/rsyslog.d/owamp-syslog.conf
+rm -f /etc/logrotate.d/perfsonar-toolkit
+
 # Set 'local5' output location to /var/log/perfsonar/owamp.log
-grep ^local5 /etc/rsyslog.d/owamp-syslog.conf &> /dev/null
+grep ^local5 /etc/rsyslog.d/30-owamp-syslog.conf &> /dev/null
 if [ $? != 0 ]; then
-cat >> /etc/rsyslog.d/owamp-syslog.conf <<EOF                                          
-# Save bwctl and owamp messages to /var/log/perfsonar/owamp.log
+cat >> /etc/rsyslog.d/30-owamp-syslog.conf <<EOF
+# Save owamp and twamp messages to /var/log/perfsonar/owamp.log
 local5.*                                                -/var/log/perfsonar/owamp.log
-EOF
-fi
-
-# Disable sync on /var/log/messages to save IO
-sed 's/ \/var\/log\/messages/ -\/var\/log\/messages/g' /etc/rsyslog.conf > /etc/rsyslog.conf.tmp
-# Cleanup any incorrect --/var/log/messages entries from earlier versions
-sed -i 's/--*\/var\/log\/messages/-\/var\/log\/messages/g' /etc/rsyslog.conf.tmp
-mv /etc/rsyslog.conf.tmp /etc/rsyslog.conf
-
-# Make sure that the owamp/bwctl log file gets rotated regularly
-grep owamp.log /etc/logrotate.d/perfsonar-toolkit &> /dev/null
-if [ $? != 0 ]; then
-cat >>/etc/logrotate.d/perfsonar-toolkit <<EOF
-/var/log/perfsonar/owamp.log {
-    sharedscripts
-    postrotate
-        /bin/kill -HUP \`cat /var/run/syslogd.pid 2> /dev/null\` 2> /dev/null || true
-        /bin/kill -HUP \`cat /var/run/rsyslogd.pid 2> /dev/null\` 2> /dev/null || true
-    endscript
-}
+& stop
 EOF
 fi
 
@@ -49,4 +34,5 @@
 # The log file needs to be created initially so that the log file isn't the
 # default of root:root and only readable by root.
 touch /var/log/perfsonar/owamp.log
-chown perfsonar:perfsonar /var/log/perfsonar/owamp.log
+chmod 664 /var/log/perfsonar/owamp.log
+chown perfsonar:adm /var/log/perfsonar/owamp.log
