Index: b/scripts/configure_firewall
===================================================================
--- a/scripts/configure_firewall	2024-04-04 08:07:09.669161362 -0400
+++ b/scripts/configure_firewall	2024-04-04 08:07:10.951265170 -0400
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
