Index: b/scripts/configure_firewall
===================================================================
--- a/scripts/configure_firewall	2023-12-08 15:42:52.308912475 -0500
+++ b/scripts/configure_firewall	2023-12-08 15:42:53.302556619 -0500
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
