Index: b/scripts/configure_firewall
===================================================================
--- a/scripts/configure_firewall	2023-04-19 15:50:39.951447915 -0400
+++ b/scripts/configure_firewall	2023-04-19 15:50:40.997377670 -0400
@@ -38,7 +38,7 @@
     if type firewall-cmd &>/dev/null; then
         firewall-cmd --quiet --runtime-to-permanent
     else
-        for iptables_cmd in "iptables" "ip6tables"; do
+        for iptables_cmd in "iptables-persistent"; do
             service $iptables_cmd save
         done
     fi
