Index: b/scripts/apache-toolkit_web_gui.conf
===================================================================
--- a/scripts/apache-toolkit_web_gui.conf	2023-05-27 17:58:13.720411000 +0000
+++ b/scripts/apache-toolkit_web_gui.conf	2023-05-31 09:35:11.113572281 +0000
@@ -11,6 +11,11 @@
 Alias /toolkit/plot /usr/lib/perfsonar/web-ng/root/admin/plot.cgi
 
 
+# Redirects requests to "/" to "/toolkit". It's done in this strange way to
+# avoid confusing people who enter an IP address and would get redirected to
+# the hostname, or vice versa.
+RedirectMatch ^/$ /toolkit/
+
 # Redirect from /index.cgi?format=json to the new host summary page
 RewriteCond %{QUERY_STRING} format=json
 RewriteRule ^/toolkit/index.cgi /toolkit/services/host.cgi?method=get_summary [R,L]
@@ -31,10 +36,8 @@
 # But, allow anonymous access via port 80 (http)
 # NOTE: Need to set non-https re-write rules and redirects here
 <VirtualHost *:80>
-    # Redirects requests to "/" to "/toolkit". It's done in this strange way to
-    # avoid confusing people who enter an IP address and would get redirected to
-    # the hostname, or vice versa.
-    RedirectMatch ^/$ /toolkit/
+    # Hide default index page
+    DocumentRoot /var/www/perfsonar
 
     # Redirect from /index.cgi?format=json to the new host summary page
     RewriteCond %{QUERY_STRING} format=json
@@ -83,23 +86,11 @@
     Options +FollowSymLinks -MultiViews +ExecCGI
     DirectoryIndex index.cgi index.php index.html
     AddHandler cgi-script .cgi
-    <IfVersion < 2.4>
-      Order allow,deny
-      Allow from all
-    </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Location>
 
 # Allow indexes on logs
@@ -113,23 +104,11 @@
     DirectoryIndex index.cgi index.php index.html
     AddHandler cgi-script .cgi
     AllowOverride All
-    <IfVersion < 2.4>
-      Order allow,deny
-      Allow from all
-    </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Directory>
 
 ##
Index: b/scripts/add_pssudo_user
===================================================================
--- a/scripts/add_pssudo_user	2023-05-27 17:58:13.720035000 +0000
+++ b/scripts/add_pssudo_user	2023-05-31 09:35:11.114920009 +0000
@@ -104,7 +104,7 @@
     fout.write(buffer)
 
 #restart ssh
-retVal = os.system("/sbin/service sshd restart")
+retVal = os.system("service ssh restart")
 if (retVal != 0):
     print(Internet2Consts.YELLOW + "Error: couldn't restart sshd. Root SSH will be disabled on next restart." + Internet2Consts.NORMAL)
     sys.exit(1)
Index: b/init_scripts/perfsonar-configure_nic_parameters
===================================================================
--- a/init_scripts/perfsonar-configure_nic_parameters	2023-05-27 18:34:41.529994256 +0000
+++ b/init_scripts/perfsonar-configure_nic_parameters	2023-05-31 09:35:30.008527000 +0000
@@ -1,17 +1,12 @@
 #!/bin/sh
-#
-# $Id$
-#
-# multi-ethtool
-#
-# chkconfig: - 95 99
-# description:	pS-Performance Toolkit script to configure the NIC parameters
-# 
-# see:
-# http://code.google.com/p/perfsonar-ps/issues/detail?id=122
-# https://bugzilla.redhat.com/show_bug.cgi?id=179877
-#
-#
+### BEGIN INIT INFO
+# Provides:          perfsonar-configure_nic_parameters
+# Required-Start:    $local_fs $remote_fs $network
+# Required-Stop:     $local_fs $remote_fs $network
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: pS-Performance Toolkit script to configure the NIC parameters
+### END INIT INFO
 
 # Source function library.
 . /etc/init.d/functions
@@ -141,6 +136,10 @@
         start
         RETVAL=$?
         ;;
+    stop)
+        # to make systemd happy
+        RETVAL=0
+        ;;
     *)
         echo $"Usage: $0 {start}"
         RETVAL=2
Index: b/init_scripts/perfsonar-generate_motd
===================================================================
--- a/init_scripts/perfsonar-generate_motd	2023-05-27 17:58:13.719685000 +0000
+++ b/init_scripts/perfsonar-generate_motd	2023-05-31 09:35:11.118120361 +0000
@@ -1,11 +1,15 @@
 #!/usr/bin/perl
-#
-# pS-Performance Toolkit script that changes the MOTD to include the toolkit
-# version number as well as the URL to go to configure the toolkit.
-#
-# chkconfig: 2345 99 99
-# description: pS-Performance Toolkit MOTD maintenance script
-#
+### BEGIN INIT INFO
+# Provides:          perfsonar-generate_motd
+# Required-Start:    $local_fs $remote_fs
+# Required-Stop:     $local_fs $remote_fs
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: pS-Performance Toolkit MOTD maintenance script
+# Description:       pS-Performance Toolkit script that changes the MOTD
+#                    to include the toolkit version number as well as
+#                    the URL to go to configure the toolkit.
+### END INIT INFO
 
 use strict;
 use warnings;
Index: b/web-ng/root/gui/reverse_traceroute.cgi
===================================================================
--- a/web-ng/root/gui/reverse_traceroute.cgi	2023-05-27 17:58:13.719452000 +0000
+++ b/web-ng/root/gui/reverse_traceroute.cgi	2023-05-31 09:35:11.122228192 +0000
@@ -273,7 +273,7 @@
 #########################################################################
 # Build the executable function for tracepath
 elsif($function eq "tracepath") {
-  $Tr="/bin/tracepath";
+  $Tr="tracepath";
   @Tropts="";
 }
 #########################################################################
Index: b/etc/perfsonar_ulimit.conf
===================================================================
--- a/etc/perfsonar_ulimit.conf	2023-05-27 17:58:13.719111000 +0000
+++ b/etc/perfsonar_ulimit.conf	2023-05-31 09:35:11.125821214 +0000
@@ -1,8 +1,8 @@
 # Increases process limit and number of files open for various perfsonar tools
 
 pscheduler  soft    nproc     32768
-apache      soft    nproc     32768
+www-data    soft    nproc     32768
 pscheduler  soft    nofile    32768
-apache      soft    nofile    32768
+www-data    soft    nofile    32768
 perfsonar   soft    nofile    32768
 owamp       soft    nofile    32768
