Index: b/bin/config_daemon.pl
===================================================================
--- a/bin/config_daemon.pl	2025-05-12 10:23:23.992786445 -0400
+++ b/bin/config_daemon.pl	2025-05-12 10:23:24.919354966 -0400
@@ -29,7 +29,7 @@
 use Log::Log4perl qw/:easy/;
 
 # set the process name
-$0 = "toolkit_config_daemon.pl";
+$0 = "configdaemon.pl";
 
 my @child_pids = ();
 
@@ -70,7 +70,7 @@
 }
 
 if ( not $PIDFILE ) {
-    $PIDFILE = "/var/run/toolkit_config_daemon.pid";
+    $PIDFILE = "/var/run/configdaemon.pid";
 }
 
 ( $status, $res ) = lockPIDFile( $PIDFILE );
Index: b/etc/configdaemon.conf
===================================================================
--- a/etc/configdaemon.conf	2025-05-12 10:23:23.993370112 -0400
+++ b/etc/configdaemon.conf	2025-05-12 10:23:24.919872633 -0400
@@ -1,6 +1,6 @@
 address     localhost
 port        9000
-firewall_script /usr/lib/perfsonar/scripts/system_environment/configure_firewall
+firewall_script /usr/lib/perfsonar/scripts/configure_firewall
 
 <access>
     <service yum_cron>
