Index: b/scripts/apache-toolkit_web_gui.conf
===================================================================
--- a/scripts/apache-toolkit_web_gui.conf	2023-12-08 15:42:52.308676976 -0500
+++ b/scripts/apache-toolkit_web_gui.conf	2023-12-08 15:42:53.373099352 -0500
@@ -11,6 +11,11 @@
 Alias /toolkit/plot /usr/lib/perfsonar/web-ng/root/admin/plot.cgi
 
+# Redirects requests to "/" to "/toolkit". It's done in this strange way to
+# avoid confusing people who enter an IP address and would get redirected to
+# the hostname, or vice versa.
+RedirectMatch ^/$ /toolkit/
+
 # Redirect from /index.cgi?format=json to the new host summary page
 RewriteCond %{QUERY_STRING} format=json
 RewriteRule ^/toolkit/index.cgi /toolkit/services/host.cgi?method=get_summary [R,L]
@@ -79,23 +84,11 @@
     Options +FollowSymLinks -MultiViews +ExecCGI
     DirectoryIndex index.cgi index.php index.html
     AddHandler cgi-script .cgi
-    <IfVersion < 2.4>
-      Order allow,deny
-      Allow from all
-    </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Location>
 
 # Allow indexes on logs
@@ -109,23 +102,11 @@
     DirectoryIndex index.cgi index.php index.html
     AddHandler cgi-script .cgi
     AllowOverride All
-    <IfVersion < 2.4>
-      Order allow,deny
-      Allow from all
-    </IfVersion>
 
-    <IfVersion >= 2.4>
-      AuthType Basic
-      AuthName "Password Required"
-      AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
-      Require valid-user
-    </IfVersion>
-    <IfVersion < 2.4>
-      AuthShadow on
-      AuthType Basic
-      AuthName "Password Required"
-      Require group psadmin
-    </IfVersion>
+    AuthType Basic
+    AuthName "Password Required"
+    AuthUserFile /etc/perfsonar/toolkit/psadmin.htpasswd
+    Require valid-user
 </Directory>
 
 ServerTokens Prod
Index: b/scripts/add_pssudo_user
===================================================================
<<<<<<< HEAD
--- a/scripts/add_pssudo_user	2023-11-21 14:26:23.220006532 +0100
+++ b/scripts/add_pssudo_user	2023-11-21 14:26:49.932802774 +0100
=======
--- a/scripts/add_pssudo_user	2023-12-08 15:42:52.308619643 -0500
+++ b/scripts/add_pssudo_user	2023-12-08 15:42:53.373425767 -0500
>>>>>>> 5.0.8
@@ -104,7 +104,7 @@
     fout.write(buffer)
 
 #restart ssh
-retVal = os.system("/sbin/service sshd restart")
+retVal = os.system("service ssh restart")
 if (retVal != 0):
     print(Internet2Consts.YELLOW + "Error: couldn't restart sshd. Root SSH will be disabled on next restart." + Internet2Consts.NORMAL)
     sys.exit(1)
Index: b/init_scripts/perfsonar-configure_nic_parameters
===================================================================
<<<<<<< HEAD
--- a/init_scripts/perfsonar-configure_nic_parameters	2023-11-21 14:26:23.222065344 +0100
+++ b/init_scripts/perfsonar-configure_nic_parameters	2023-11-21 14:26:49.933422981 +0100
=======
--- a/init_scripts/perfsonar-configure_nic_parameters	2023-12-08 15:42:52.307580065 -0500
+++ b/init_scripts/perfsonar-configure_nic_parameters	2023-12-08 15:42:53.373624557 -0500
>>>>>>> 5.0.8
@@ -1,17 +1,12 @@
 #!/bin/sh
-#
-# $Id$
-#
-# multi-ethtool
-#
-# chkconfig: - 95 99
-# description:	pS-Performance Toolkit script to configure the NIC parameters
-# 
-# see:
-# http://code.google.com/p/perfsonar-ps/issues/detail?id=122
-# https://bugzilla.redhat.com/show_bug.cgi?id=179877
-#
-#
+### BEGIN INIT INFO
+# Provides:          perfsonar-configure_nic_parameters
+# Required-Start:    $local_fs $remote_fs $network
+# Required-Stop:     $local_fs $remote_fs $network
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: pS-Performance Toolkit script to configure the NIC parameters
+### END INIT INFO
 
 # Source function library.
 . /etc/init.d/functions
@@ -141,6 +136,10 @@
         start
         RETVAL=$?
         ;;
+    stop)
+        # to make systemd happy
+        RETVAL=0
+        ;;
     *)
         echo $"Usage: $0 {start}"
         RETVAL=2
Index: b/init_scripts/perfsonar-generate_motd
===================================================================
<<<<<<< HEAD
--- a/init_scripts/perfsonar-generate_motd	2023-11-21 14:26:23.223683072 +0100
+++ b/init_scripts/perfsonar-generate_motd	2023-11-21 14:26:49.934235963 +0100
=======
--- a/init_scripts/perfsonar-generate_motd	2023-12-08 15:42:52.307638523 -0500
+++ b/init_scripts/perfsonar-generate_motd	2023-12-08 15:42:53.373841348 -0500
>>>>>>> 5.0.8
@@ -1,11 +1,15 @@
 #!/usr/bin/perl
-#
-# pS-Performance Toolkit script that changes the MOTD to include the toolkit
-# version number as well as the URL to go to configure the toolkit.
-#
-# chkconfig: 2345 99 99
-# description: pS-Performance Toolkit MOTD maintenance script
-#
+### BEGIN INIT INFO
+# Provides:          perfsonar-generate_motd
+# Required-Start:    $local_fs $remote_fs
+# Required-Stop:     $local_fs $remote_fs
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: pS-Performance Toolkit MOTD maintenance script
+# Description:       pS-Performance Toolkit script that changes the MOTD
+#                    to include the toolkit version number as well as
+#                    the URL to go to configure the toolkit.
+### END INIT INFO
 
 use strict;
 use warnings;
Index: b/web-ng/root/gui/reverse_traceroute.cgi
===================================================================
<<<<<<< HEAD
--- a/web-ng/root/gui/reverse_traceroute.cgi	2023-11-21 14:26:23.225381920 +0100
+++ b/web-ng/root/gui/reverse_traceroute.cgi	2023-11-21 14:26:49.935167292 +0100
=======
--- a/web-ng/root/gui/reverse_traceroute.cgi	2023-12-08 15:42:52.323786984 -0500
+++ b/web-ng/root/gui/reverse_traceroute.cgi	2023-12-08 15:42:53.374161180 -0500
>>>>>>> 5.0.8
@@ -273,7 +273,7 @@
 #########################################################################
 # Build the executable function for tracepath
 elsif($function eq "tracepath") {
-  $Tr="/bin/tracepath";
+  $Tr="tracepath";
   @Tropts="";
 }
 #########################################################################
Index: b/etc/perfsonar_ulimit.conf
===================================================================
<<<<<<< HEAD
--- a/etc/perfsonar_ulimit.conf	2023-11-21 14:26:23.227261286 +0100
+++ b/etc/perfsonar_ulimit.conf	2023-11-21 14:26:49.936516741 +0100
=======
--- a/etc/perfsonar_ulimit.conf	2023-12-08 15:42:52.307127942 -0500
+++ b/etc/perfsonar_ulimit.conf	2023-12-08 15:42:53.374454928 -0500
>>>>>>> 5.0.8
@@ -1,8 +1,8 @@
 # Increases process limit and number of files open for various perfsonar tools
 
 pscheduler  soft    nproc     32768
-apache      soft    nproc     32768
+www-data    soft    nproc     32768
 pscheduler  soft    nofile    32768
-apache      soft    nofile    32768
+www-data    soft    nofile    32768
 perfsonar   soft    nofile    32768
 owamp       soft    nofile    32768
