Index: b/scripts/nptoolkit-configure.py
===================================================================
--- a/scripts/nptoolkit-configure.py	2025-05-20 12:19:25.091968849 -0400
+++ b/scripts/nptoolkit-configure.py	2025-05-20 12:19:28.033172025 -0400
@@ -99,7 +99,7 @@
                     },
                     {
                          "description": "Change Timezone",
-                         "command": "/usr/bin/tzselect",
+                         "command": "/usr/sbin/dpkg-reconfigure tzdata",
                          "isConfiguredFunction": None,
                          "required": False,
                          "requiresReboot": True,
@@ -130,7 +130,7 @@
 
     for item in known_items:
         # Skip the item if it's associated command isn't available.
-        if (not os.path.exists(item["command"])):
+        if (not os.path.exists(item["command"].split(" ")[0])):
             continue
 
         item = toolkitItem(item["description"], item["command"], item["isConfiguredFunction"], item["required"], item["requiresReboot"])
